application/sitebuilder/build.py

```diff
@@ -18,7 +18,7 @@
 
 def do_it(application, build):
     with application.app_context():
-
+        print(f"Building static site in environment: {application.config['ENVIRONMENT']}")
```

Comment:
Remove debug line?

Reply:
I added this on purpose... just for future proofing ;)